package.json

```diff
@@ -16,6 +16,8 @@
   "scripts": {
     "build": "ember build --environment=production",
     "lint": "concurrently \"npm:lint:*(!fix)\" --names \"lint:\"",
+    "lint:css": "stylelint --allow-empty-input \"**/*.css\"",
```

Comment:
This doesn't actually do anything yet since we don't have .css files in the repo. I'll create a separate PR to set it up for .scss similar to this one: https://github.com/lblod/frontend-lpdc/pull/6